src/suites/cts/validation/setBindGroup.spec.ts

```diff
@@ -0,0 +1,193 @@
+export const description = `
+setBindGroup validation tests.
+`;
+
+import { TestGroup } from '../../../framework/index.js';
+import GLSL from '../../../tools/glsl.macro.js';
+
+import { ValidationTest } from './validation_test.js';
+
+export class F extends ValidationTest {
+  setup(): { bindGroup: GPUBindGroup; pipelineLayout: GPUPipelineLayout } {
+    // Dynamic buffer offsets require offset to be divisible by 256
+    const MIN_DYNAMIC_BUFFER_OFFSET_ALIGNMENT = 256;
+    const BINDING_SIZE = 9;
+
+    const bindGroupLayout = this.device.createBindGroupLayout({
+      bindings: [
+        {
+          binding: 0,
+          visibility: GPUShaderStage.COMPUTE | GPUShaderStage.FRAGMENT,
+          type: 'uniform-buffer',
+          dynamic: true,
+        },
+        {
+          binding: 1,
+          visibility: GPUShaderStage.COMPUTE | GPUShaderStage.FRAGMENT,
+          type: 'storage-buffer',
+          dynamic: true,
+        },
+      ],
+    });
+
+    const uniformBuffer = this.device.createBuffer({
+      size: 2 * MIN_DYNAMIC_BUFFER_OFFSET_ALIGNMENT + 8,
+      usage: GPUBufferUsage.UNIFORM,
+    });
+
+    const storageBuffer = this.device.createBuffer({
+      size: 2 * MIN_DYNAMIC_BUFFER_OFFSET_ALIGNMENT + 8,
+      usage: GPUBufferUsage.STORAGE,
+    });
+
+    const bindGroup = this.device.createBindGroup({
+      bindings: [
+        {
+          binding: 0,
+          resource: {
+            buffer: uniformBuffer,
+            size: BINDING_SIZE,
+          },
+        },
+        {
+          binding: 1,
+          resource: {
+            buffer: storageBuffer,
+            size: BINDING_SIZE,
+          },
+        },
+      ],
+      layout: bindGroupLayout,
+    });
+
+    const pipelineLayout = this.device.createPipelineLayout({
+      bindGroupLayouts: [bindGroupLayout],
+    });
+
+    return { bindGroup, pipelineLayout };
+  }
+
+  testComputePass(dynamicOffsets: number[]): void {
+    const { bindGroup, pipelineLayout } = this.setup();
+
+    const module = this.device.createShaderModule({
```

Comment:
A compute pass with just a setBindGroup should still fail so the module isn't needed here (nor is the pipeline and dispatch call)

Reply:
Done.